docs/first-steps/post-deployment.md

```diff
@@ -20,16 +20,23 @@ output "user_natasha_romanoff_login_profile_encrypted_password" {
 }
 ```
 
-Then, in the `global/base-identities` directory, run:
+Then, in the `global/base-identities` directory, run the output command with the `-json` flag:
 
 ``` bash
-leverage terraform apply
-leverage terraform output
+leverage terraform output -json
 ```
 ```
 ...
-user_natasha_romanoff_login_profile_encrypted_password = "SipVOzVtNTI0Ml...EZmJFxxQSteYQ=="
-user_natasha_romanoff_name = "natasha.romanoff"
+"user_natasha_romanoff_name": {
+  "sensitive": false,
+  "type": "string",
+  "value": "natasha.romanoff"
+},
+"user_natasha_romanoff_login_profile_encrypted_password": {
+  "sensitive": true,
+  "type": "string",
+  "value": "wcDMAyRZJTaxw5v1AQwAy6c...............2mBIbNFxF1Tp/ilvyk8eEHvAA="
+}
```

Comment:
@rodriguez-matias Matu, I'm wondering why you've changed this command with the json output instead of the original one

CC: @angelofenoglio @juanmatias

Reply:
@exequielrafaela  with the original method, in order to get the temp credentials , we need to do:
- change the sensitive argument from true to false .
- Save the changes and then run the apply + output commands
- Then change it back again to sensitive = True
- And apply again the changes

with the output -json command we can obtain the value of the temp passwd without make any change in the code